cogs/moderation/moderation.py

```diff
@@ -44,9 +44,10 @@ async def _kick(self, ctx: Context, member: discord.Member, *, reason="Breaking
             await ctx.send(embed=embed)
 
             await self.bot.db.execute(
-                '''INSERT INTO Cases(id, guildid, userid, modid, username, modname, case_type, created_at)
-                VALUES ($1, $2, $3, $4, $5, $6, $7, $8)''', next(self.bot.idgen), ctx.guild.id, member.id, ctx.author.id, member,
-                ctx.author, "kick", curtime)
+                '''INSERT INTO Cases(id, guildid, userid, modid, username, modname, case_type, created_at, case_data)
+                VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)''', next(self.bot.idgen), ctx.guild.id, member.id,
+                ctx.author.id, member,
```

Comment:
Does this work? I have a feeling it'll error when you try inserting `member` as it is a `discord.Member` object